patches/minecraft/net/minecraft/potion/EffectType.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/potion/EffectType.java
++++ b/net/minecraft/potion/EffectType.java
+@@ -15,7 +15,6 @@
+       this.field_220307_d = p_i50390_3_;
+    }
+ 
+-   @OnlyIn(Dist.CLIENT)
```

Comment:
Why does the server need to know the effect color?

Reply:
So can directly construct a chat message that includes the Potion formatting (ex Removed *Hunger Effect*) server side.

This prevents needs to make a custom channel+packet to send a message to have the client construct the message.